blazor/pdfviewer/how-to/prevent-scrolling.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: Prevent the PDF from scrolling | Syncfusion
+description: Learn here all about how to prevent the PDF from scrolling and remove the vertical scrollbar in Syncfusion Blazor PDF Viewer component and more.
+platform: Blazor
+control: PDF Viewer
+documentation: ug
+---
+
+# Prevent the PDF from scrolling and remove the vertical scrollbar
+
+To prevent the PDF from scrolling and remove the vertical scroll bar in the Syncfusion Blazor PDF Viewer component, you can use CSS to set the `overflow` property of the component container to `hidden`.

```

Comment:
To prevent a PDF from scrolling and remove the vertical scroll bar in the Syncfusion Blazor PDF Viewer component, use CSS to set the `overflow` property of the component container to `hidden.`